docs/sphinx/examples/cpp/basics/expectation_values.cpp

```diff
@@ -21,9 +21,10 @@ struct ansatz {
 int main() {
 
   // Build up your spin op algebraically
-  using namespace cudaq::spin;
-  cudaq::spin_op h = 5.907 - 2.1433 * x(0) * x(1) - 2.1433 * y(0) * y(1) +
-                     .21829 * z(0) - 6.125 * z(1);
+  cudaq::spin_op h =
+      5.907 - 2.1433 * cudaq::spin_op::x(0) * cudaq::spin_op::x(1) -
+      2.1433 * cudaq::spin_op::y(0) * cudaq::spin_op::y(1) +
+      .21829 * cudaq::spin_op::z(0) - 6.125 * cudaq::spin_op::z(1);
```

Comment:
nit: there is another similar code in `docs/sphinx/examples/cpp/other/remote_sim/simple.cpp` that needs to be updated.